<launch>
	<!-- Simulation Webots -->
  	<include file="$(find webots_natasha)/launch/start_simulation.launch"/>

	<!-- Movement -->
	<include file="$(find movecreator_qt)/launch/natasha_movecreator.launch"></include>
    
	<!-- Arduino
	<node name="arduino" pkg="rosserial_python" type="serial_node.py" output="screen">
		<param name="port" value="/dev/ttyACM0" />
		<param name="baud" value="57600" />
	</node>-->

	<!-- Actions -->
	<node name="yamlTransition" pkg="state_machine" type="yamlTransition.py" output="screen" >   </node>
	<node name="brainGoalie" pkg="state_machine" type="brainGoalie.py" output="screen" >    </node>

	<!-- Vision -->
	<!--node name="Vision" pkg="object_finder" type="connecting_and_showing_current_frame_robocup.py" output="screen">    </node-->
	<!--node name="Behaviour_test" pkg="ballsim" type="Behaviour_test.py" output="screen">    </node-->

</launch>
